app/components/search-facet-taxonomy.js

```diff
@@ -86,6 +86,9 @@ export default Ember.Component.extend(Analytics, {
                     })
                 });
                 this._expandMany(items);
+                //Only auto-expand if no subjects are selected.
+                if (!items.length > 0)
```

Comment:
A couple of things here.  

1. Would it be better to write this out as `if (items.length === 0)`.  It might make it a bit clearer to other people reading it out.

2. I think the preferred way to write this statement out would be with the curly braces.